src/integrators/integrator_interface.jl

```diff
@@ -332,3 +332,33 @@ end
 @inline function DiffEqBase.get_du!(out,integrator::SDEIntegrator)
   @. out = (integrator.u - integrator.uprev) / integrator.dt
 end
+
+function DiffEqBase.set_t!(integrator::SDEIntegrator, t::Real)
+  if integrator.opts.save_everystep
+    error("Integrator time cannot be reset unless it is initialized",
+          " with save_everystep=false")
+  end
+  if !isdtchangeable(integrator.alg)
```

Comment:
There was `alg_extrapolates(integrator.alg) ||` in the ODE version. Anything like that in SDE?

Reply:
Nope, there's no extrapolating in the SDEs.

Reply:
No other traits that prevent non-`reinit!` version of `set_t!`?

Reply:
I don't think there is. Comment out the `isdtchangable` though since that could show up if a multistep SDE method is implemented, which I do hope to get to.

Reply:
The algorithm traits in OrdinaryDiffEq.jl were added as needed, but now that we kind of have a good set it may be useful to up that to some more general DiffEq thing.

Reply:
> Comment out the isdtchangable

What do you mean?  Just use `integrator.t = t` only?

I see that `isdtchangable` always returns true (except the methods in BridgeDiffEq.jl?).  So, maybe you mean you don't want a "dead path" in the function?  Comment out the `reinit!` path until you implement other methods?

Reply:
oh, I didn't realize `isdtchangable` for SDEs already exists haha. I guess I do have it there just for the possible multistep methods in the future, so it's fine.

Reply:
I see!